webapp/src/routes/my/services-and-products/_wallets/wallet-form.svelte

```diff
@@ -48,94 +46,218 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 			conformance_checks: z.array(ConformanceCheckSchema).nullable()
 		});
 
-	const form = createForm<z.infer<typeof schema>>({
+	async function autoPopulateFromUrl() {
+		if (!autoPopulateUrl.trim()) return;
+
+		isProcessingWorkflow = true;
+		autoPopulateError = '';
+
+		try {
+			const response = await pb.send('/wallet/start-check', {
+				method: 'POST',
+				body: {
+					WalletURL: autoPopulateUrl.trim()
+				}
+			});
+
+			if (response) {
+				if (response.logo) {
+					response.logo_url = response.logo;
+					delete response.logo;
+				}
```

Comment:
safe? I see that there is in the wallet page, that has some logic to choose between the two fields... do we want to have this kind of attributes replacement? I suspect, that unify everything in a single field, should be better...